cmd/agent/subcommands/run/command.go

```diff
@@ -406,7 +406,7 @@ func startAgent(
 	telemetry telemetry.Component,
 	_ sysprobeconfig.Component,
 	server dogstatsdServer.Component,
-	serverDebug dogstatsddebug.Component,
+	_ dogstatsddebug.Component,
```

Comment:
I suppose, at a time, we will need to remove completely those parameters rather than using blank identifier ? @ogaca-dd